code/iaas/model/src/main/java/io/cattle/platform/core/cleanup/TableCleanup.java

```diff
@@ -152,7 +178,9 @@ public void run() {
         Date otherCutoff = new Date(current - MAIN_TABLES_AGE_LIMIT_SECONDS.getValue() * SECOND_MILLIS);
         cleanupLabelTables(otherCutoff);
         cleanupExternalHandlerExternalHandlerProcessMapTables(otherCutoff);
-        cleanupMountTable(otherCutoff);
+        cleanupTableByQuery(MOUNT_DELETE_QUERY, "mount", otherCutoff);
+        cleanupTableByQuery(SECRET_CLEANUP_VSPM_QUERY, "volume_storage_pool_map", otherCutoff);
+        cleanupTableByQuery(SECRET_CLEAUP_VOLUME_QUERY, "volume", otherCutoff);
```

Comment:
Add the new queries at the end of the cleanup run?

Reply:
@prachidamle moved